lxd/db/cluster/nodes_cluster_groups.mapper.go

```diff
@@ -222,26 +224,25 @@ func NodeClusterGroupExists(ctx context.Context, tx *sql.Tx, groupID int) (bool,
 	return true, nil
 }
 
-// GetNodeClusterGroupID return the ID of the node_cluster_group with the given key.
-// generator: node_cluster_group ID
-func GetNodeClusterGroupID(ctx context.Context, tx *sql.Tx, groupID int) (int64, error) {
-	stmt, err := Stmt(tx, nodeClusterGroupID)
+// GetNodeClusterGroup returns the node_cluster_group entry with the given group ID and node name.
+// generator: node_cluster_group GetOne
+func GetNodeClusterGroup(ctx context.Context, tx *sql.Tx, groupID int, nodeName string) (*NodeClusterGroup, error) {
```

Comment:
you cannot modify any .mapper.go files as they are generated by the DB generator.

Reply:
instead it sounds like the definition of the table isnt correct in the metadata, here: 

https://github.com/canonical/lxd/blob/main/lxd/db/cluster/cluster_groups.go#L7-L25

Or we've hit a limitation with the DB generator.

Cc @masnax

Reply:
Thanks @masnax! I've made the suggested update

Reply:
Looking into this more, it seems in other places in the code base we use `tx.AddNodeToClusterGroup`, so can we instead of updating the generator rules use this function in `clusterGroupsPost` and `clusterGroupPatch`.

Potentially then we can remove some of the generated DB functions for this table entirely.
And then we a consistent way to add cluster members to a cluster group.

Thanks!

Reply:
@tomponline thanks, I've updated `clusterGroupsPost` and `clusterGroupPatch` to use `tx.AddNodeToClusterGroup` instead of the generated `CreateNodeClusterGroup`. I've also removed the now-unneeded generated functions for the `nodes_cluster_groups` table.